tmva/tmva/test/rreader.cxx

```diff
@@ -53,6 +53,7 @@ void TrainClassificationModel()
    factory->BookMethod(dataloader, TMVA::Types::kBDT, "BDT", "!V:!H:NTrees=100:MaxDepth=2");
    factory->TrainAllMethods();
    output->Close();
+   delete data;
```

Comment:
I think wrapping the file in a unique pointer would be the preferred solution, like this:
```c++
std::unique_ptr<data>{TFile::Open(filename.c_str())};
```

Reply:
Done, thanks for the suggestion